src/app/components/ClickOnMe/ClickOnMeTitle.tsx

```diff
@@ -0,0 +1,11 @@
+export const ClickOnMeTitle = () => {
+    return (
+      <div className="mb-4 flex justify-center">
+      <div className="desktop:mb-[60px] mb-[26px]">
+        <h2 className="text-26 desktop:mb-[8px] desktop:text-36 mb-[10px] font-bold leading-[1.2] tracking-[-0.02em]">
+          Kliknij na mnie
+        </h2>
+      </div>
+      </div>
+    );
+  };
```

Comment:
It would be nice if you type a component and add a text props to it and then add the import to the main component. Also try to make such changes to the other smaller components: Here is an example of how to do it:

```
import React from 'react';

interface ClickOnMeTitleProps {
  text: string; // Typ dla prop 'text'
}

export const ClickOnMeTitle: React.FC<ClickOnMeTitleProps> = ({ text }) => {
  return (
    <div className="mb-4 flex justify-center">
      <div className="desktop:mb-[60px] mb-[26px]">
        <h2 className="text-26 desktop:mb-[8px] desktop:text-36 mb-[10px] font-bold leading-[1.2] tracking-[-0.02em]">
          {text} {/* Użyj prop 'text' */}
        </h2>
      </div>
    </div>
  );
};
```

And then in ClickOnMe.tsx you add along with the import.

`<ClickOnMeTitle text="Kliknij na mnie" />`


Changing components to type in TypeScript has several important benefits that can increase the quality and stability of your code. Here are some reasons why you should do so:

Typing:

Defining types for props allows you to better understand what data should be passed to the component. It also makes troubleshooting easier, as errors related to types are caught during compilation rather than at runtime.
Documentation:

Types act as a form of documentation. Other developers (or you yourself in the future) can more easily understand what data is required by the component, making the code more readable.

Refactoring:

When components are well typed, refactoring is easier. Changes to data types automatically inform you of places in your code that need to be updated.
Error avoidance:

Types can help catch potential errors, such as passing the wrong data types to components, which can lead to execution errors.

Reusability:

Typed components are more flexible and can be more easily used in different contexts because they are clear in their requirements for props.